Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM python:3.7-slim
 MAINTAINER cypherpunkarmory
-
+RUN apt-get update && apt-get install git -y
```

Comment:
This is here because zappa hasnt merged the pr adding support for python 3.7
So i need to have git to pip install my fork